hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/BackendResolverTest.java

```diff
@@ -166,7 +166,9 @@ public void checkBackendEntityGeneratedFromHttpEventType1() {
 
     Map<String, AttributeValue> attributes = backendInfo.getAttributes();
     assertEquals(
-        Map.of("BACKEND_DESTINATION", AttributeValueCreator.create("/path/12314/?q=ddds#123")),
+        Map.of(
+            "BACKEND_DESTINATION",
+            AttributeValueCreator.create("/product/5d644175551847d7408760b1")),
```

Comment:
Can you add one more test or modify any other test to also check for fallback attributes?

Reply:
sure, let me do that.

Reply:
cool.